libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -231,6 +250,70 @@ export class Simulator {
     this._running = true;
   }
 
+  /**
+   * Start a server that allows any resource to be accessed via HTTP.
+   */
+  public async startServer(): Promise<void> {
+    // import lazily to reduce SDK import time (20ms or so)
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    const express_: typeof express = require("express");
```

Comment:
Since this is an async function you can do a dynamic import. Also I don't think the top level type import or annotation here is necessary (maybe it is, not sure)

```suggestion
    const express = await import("express");
```

Reply:
Ah good catch, I forgot I was in async :)